src/components/AccountList.tsx

```diff
@@ -339,6 +339,9 @@ const getSortingFunction = (sortKey: ISortTypes): TSortFunction => {
   }
 };
 
+export const screenIsMobileSized = (): boolean =>
+  window.matchMedia(`(max-width: ${breakpointToNumber(BREAK_POINTS.SCREEN_XS)}px)`).matches;
+
```

Comment:
we have `isMobile` from `useScreenSize` for this

Reply:
It has to match the breakpoint for the table. I have simplified.